plugins/redis/README.md

```diff
@@ -85,6 +85,12 @@ Expire(CONST VARSTRING key, CONST VARSTRING options, UNSIGNED database = 0, UNSI
 INTEGER DBSize(CONST VARSTRING options, UNSIGNED database = 0, CONST VARSTRING password = '', UNSIGNED timeout = 1000)
 ```
 
+###PUB-SUB
+```
+EXPORT Publish(VARSTRING keyOrChannel, STRING message, INTEGER4 database = 0, BOOLEAN lockedKey = FALSE) := redis.Publish(keyOrChannel, message, options, database, password, timeout, lockedKey);
```

Comment:
timeout and paswsword aren't in the parameters.
Also, not consistent with the definitions above which are just the prototypes.